documentation/modules/adding-source-provider.adoc

```diff
@@ -83,34 +83,71 @@ endif::[]
+
+.. In the {ocp} web console, click *Migration for Virtualization > Providers*.
+... Click *Create Provider*.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
++
+If you have Administrator privileges, you can see all projects, otherwise, you can see only the projects you are authorized to work with.
+
+... Click *{vmw}*.
+
+.. If you have Administrator privileges, in the {ocp} web console, click *Migration for Virtualization > Overview*.
+... In the *Welcome* pane, click *{vmw}*.
++ 
+If the *Welcome* pane is not visible, click *Show the welcome card* in the upper-right corner of the page, and click *{vmw}* when the *Welcome* pane opens.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
++
+If you have Administrator privileges, you can see all projects, otherwise, you can see only the projects you are authorized to work with.
 
+. Specify the following fields:
 +
 *Provider details*
 
 * *Provider resource name*: Name of the source provider.
 * *Endpoint type*: Select the vSphere provider endpoint type. Options: *vCenter* or *ESXi*. You can migrate virtual machines from vCenter, an ESX/ESXi server that is not managed by vCenter, or from an ESX/ESXi server that is managed by vCenter but does not go through vCenter.
 * *URL*: URL of the SDK endpoint of the vCenter on which the source VM is mounted. Ensure that the URL includes the `sdk` path, usually `/sdk`. For example, `https://vCenter-host-example.com/sdk`. If a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate.
 * *VDDK init image*: `VDDKInitImage` path. It is strongly recommended to create a VDDK init image to accelerate migrations. For more information, see xref:../master.adoc#creating-vddk-image_mtv[Creating a VDDK image].
-
-
 +
 *Provider credentials*
 
 * *Username*: vCenter user or ESXi user. For example, `user@vsphere.local`.
 * *Password*: vCenter user password or ESXi user password.
 
-[start=5]
+[start=3]
 include::snip-certificate-options.adoc[]
 
 endif::[]
 
 ifdef::rhv[]
-. Click *Red Hat Virtualization*
+. Access the *Create provider* page for {rhv-full} by doing one of the following:
+
+.. In the {ocp} web console, click *Migration for Virtualization > Providers*.
+... Click *Create Provider*.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
```

Comment:
```suggestion
If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the same as the active project.
```